source/common/upstream/outlier_detection_impl.h

```diff
@@ -290,6 +290,7 @@ class DetectorConfig {
   uint64_t consecutive5xx() const { return consecutive_5xx_; }
   uint64_t consecutiveGatewayFailure() const { return consecutive_gateway_failure_; }
   uint64_t maxEjectionPercent() const { return max_ejection_percent_; }
+  bool mayAlwaysEjectOne() const { return always_eject_one_host_; }
```

Comment:
For consistency the name of the method should also be changed.

Reply:
done. Do we need a release note?

Reply:
yes. Please add reference to a new api field and describe that it overwrites max allowed percentage.